sw/device/lib/testing/test_framework/check.h

```diff
@@ -258,14 +258,18 @@
  *
  * @param expr An expression which evaluates to a `status_t`.
  */
-#define UNWRAP(expr, ...)                          \
-  ({                                               \
-    status_t status_ = expr;                       \
-    if (!status_ok(status_)) {                     \
-      LOG_ERROR("CHECK-STATUS-fail: %r", status_); \
-      test_status_set(kTestStatusFailed);          \
-    }                                              \
-    status_.value;                                 \
+#define UNWRAP(expr, ...)                                \
+  ({                                                     \
+    status_t status_ = expr;                             \
+    if (!status_ok(status_)) {                           \
+      if (kDeviceLogBypassUartAddress) {                 \
+        LOG_ERROR("CHECK-STATUS-fail: 0x%08x", status_); \
```

Comment:
or update the following: https://cs.opensource.google/opentitan/opentitan/+/master:util/device_sw_utils/extract_sw_logs.py